plugins/in_calyptia_fleet/in_calyptia_fleet.c

```diff
@@ -470,11 +472,16 @@ static void *do_reload(void *data)
 
     flb_free(reload);
     sleep(5);
-#ifndef FLB_SYSTEM_WINDOWS
-    kill(getpid(), SIGHUP);
-#else
-    GenerateConsoleCtrlEvent(1 /* CTRL_BREAK_EVENT_1 */, 0);
-#endif
+
+    ctx_signal = FLB_CTX_SIGNAL_RELOAD;
```

Comment:
Please encapsulate all of these signaling code blocks in small inline functions and reuse them.